tools/tertiary-analysis/scanpy/scanpy-run-fdg.xml

```diff
@@ -47,9 +47,7 @@ PYTHONIOENCODING=utf-8 scanpy-cli embed fdg
         <param name="neighbors_key" argument="--neighbors-key" value="neighbors" type="text"
                label="Name of the slot that holds the KNN graph"/>
         <param name="obsp" type="boolean" checked="false" label="Use .obsp[obsp] as adjacency" help="You can’t specify both obsp and neighbors-key at the same time."/>
-        <param name="init_pos" argument="--init-pos" type="text" label="Method to initialise embedding, any key for adata.obsm or choose from the preset methods">
-          <option value="paga">paga</option>
-        </param>
+        <param name="init_pos" argument="--init-pos" type="text" label="Method to initialise embedding, any key for adata.obsm or choose from the preset methods"/>
```

Comment:
Is the `present methods` part still applicable? or that is unrelated to the `paga` value being removed?

Reply:
Unrelated. I think this is just malformed field that wasn't noticed until some Scanpy change highlighted it.